tasks/main.yml

```
@@ -46,96 +82,139 @@
     state: directory
     mode: "0755"
 
-- name: Generate a configuration for kernel settings
-  template:
-    src: "{{ __kernel_settings_profile_src }}.j2"
-    dest: "{{ __kernel_settings_profile_filename }}"
-    force: false
-    mode: "0644"
+- name: Get active_profile
+  slurp:
+    path: "{{ __kernel_settings_tuned_active_profile }}"
+  register: __kernel_settings_tuned_current_profile
+
+- name: Set active_profile
+  set_fact:
+    __kernel_settings_active_profile: "{{ __active_profile }}"
+  vars:
+    __cur_profile: "{{ __kernel_settings_tuned_current_profile.content |
+      b64decode | trim }}"
+    __active_profile: "{{ __cur_profile
+      if __kernel_settings_tuned_profile in __cur_profile
+      else __cur_profile ~ ' ' ~ __kernel_settings_tuned_profile }}"
+
+- name: Ensure kernel_settings is in active_profile
+  copy:
+    content: >
+      {{ __kernel_settings_active_profile }}
+    dest: "{{ __kernel_settings_tuned_active_profile }}"
+    mode: preserve
+  register: __kernel_settings_register_profile
+
+- name: Set profile_mode to manual
+  copy:
+    content: >
```

```suggestion
    content: manual
```
is it the same thing?

---

I want to ensure the file ends with a newline - not sure if `content: manual` will ensure that